pysam/libcalignedsegment.pyx

```diff
@@ -999,6 +999,10 @@ cdef class AlignedSegment:
         <,=,> to *other*
         '''
 
+        # avoid segfault when other equals None
+        if other is None:
+            return -1
```

Comment:
Im not certain what the right return value should have been here, but `__richcmp__` (the caller that I saw in this file) seems like it doesn't care as long as it is non-zero.

Reply:
I'm looking into this now.